packages/@okta/vuepress-site/guides/sign-into-mobile-app/configure-packages/iOS/installsdk.md

```diff
@@ -1,10 +1,12 @@
-Use the library [OktaOidc](https://github.com/okta/okta-oidc-ios), available through [CocoaPods](http://cocoapods.org/pods/OktaOidc). To install it, add the following line to your Podfile:
+Use the library [OktaOidc](https://github.com/okta/okta-oidc-ios), available through [CocoaPods](http://cocoapods.org/pods/OktaOidc).
```

Comment:
Use the library [Okta OIDC](https://github.com/okta/okta-oidc-ios) that is available through [CocoaPods](http://cocoapods.org/pods/OktaOidc).